src/EventListener/ReadListener.php

```diff
@@ -56,10 +59,12 @@ public function onKernelRequest(GetResponseEvent $event)
             return;
         }
 
-        if (isset($attributes['collection_operation_name'])) {
-            $data = $this->getCollectionData($request, $attributes);
-        } else {
-            $data = $this->getItemData($request, $attributes);
+        foreach (OperationTypes::TYPES as $operationType) {
+            if (isset($attributes["{$operationType}_operation_name"])) {
```

Comment:
Brackets make it easier to read and is (of course very slightly) faster to boot.